libs/community/langchain_community/vectorstores/vectara.py

```diff
@@ -28,29 +34,30 @@ class SummaryConfig:
     """
 
     is_enabled: bool = False
-    max_results: int = 7
+    max_results: int = 5
     response_lang: str = "eng"
-    prompt_name: str = "vectara-summary-ext-v1.2.0"
+    stream: bool = True
+    prompt_name: str = "vectara-summary-ext-24-05-sml"
 
 
 @dataclass
-class MMRConfig:
```

Comment:
I believe this is a breaking change for anyone currently using `MMRConfig` - could we add some kind of backwards compatibility version that allows the old interface as well?

Reply:
See your broader comment on breaking change. New update should reflect support for backwards compatibility.